cmd/osm-controller/osm-controller.go

```diff
@@ -172,8 +169,6 @@ func main() {
 
 	endpointsProviders := []endpoint.Provider{provider}
 
-	// TODO (#88): Add Azure Endpoint provider to list of providers when supported
```

Comment:
We should have left a comment around adding new providers in light of issue #88 tracking this.